common/src/GtinHandler.php

```diff
@@ -33,16 +36,23 @@ public function getGtin(): ?string {
         if (is_plugin_active('woocommerce-product-feeds/woocommerce-gpf.php')) {
             return $this->handleGpf();
         }
-        return $this->getGtinFromMeta($this->getGtinMetaKey());
+        foreach (
+            array_filter(self::SUPPORTED_PLUGINS, [$this, 'filterMetaPlugins'])
```

Comment:
You don't need a callback function for array_filter if you want to filter on falseness.  You can just do `array_filter(self::SUPPORTED_PLUGINS)`.

But maybe it's even simpler to just do this on the next line:

~~~
if ($keys && is_plugin_active($plugin_name))
~~~

Reply:
oh wow